osu.Framework/Platform/SDL3/SDL3Window_Windowing.cs

```diff
@@ -122,22 +122,10 @@ private void initialiseWindowingAfterCreation()
             WindowMode.TriggerChange();
         }
 
-        private bool focused;
-
         /// <summary>
         /// Whether the window currently has focus.
         /// </summary>
-        public bool Focused
-        {
-            get => focused;
-            protected set
-            {
-                if (value == focused)
-                    return;
-
-                isActive.Value = focused = value;
-            }
-        }
+        public bool Focused { get; private set; }
```

Comment:
```suggestion
        public bool Focused { get; protected set; }
```

To fix Android build.

Reply:
I'd rather fix this by updating android code if we can.

Reply:
@Susko3 can you test the android change i committed?